src/Goteo/Repository/DataSetRepository.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/*
+ * This file is part of the Goteo Package.
+ *
+ * (c) Platoniq y Fundación Goteo <fundacion@goteo.org>
+ *
+ * For the full copyright and license information, please view the README.md
+ * and LICENSE files that was distributed with
+ */
+
+ namespace Goteo\Repository;
+
+ use Goteo\Application\Exception\ModelException;
+ use Goteo\Application\Exception\ModelNotFoundException;
+ use Goteo\Model\DataSet;
+ use PDO;
+ use PDOException;
+
+ class DataSetRepository extends BaseRepository {
+     protected ?string $Table = 'data_set';
```

Comment:
If this property is not going to be used, we better remove it. The same in the `BaseRepository` class.